### PR TITLE
Add point in time APIs

### PR DESCRIPTION
**Pull request by stevejgordon**
This adds support for the new open/close point in time APIs.

Contributes to #5096

### CODE REVIEW

File: tests/Tests/Search/PointInTime/Close/ClosePointInTimeApiTests.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+using FluentAssertions;
+using Nest;
+using Tests.Core.Extensions;
+using Tests.Core.ManagedElasticsearch.Clusters;
+using Tests.Framework.EndpointTests;
+using Tests.Framework.EndpointTests.TestState;
+
+namespace Tests.Search.PointInTime.Close
+{
+	public class ClosePointInTimeApiTests
```
Comment (Mpdreamz) [inline tests/Tests/Search/PointInTime/Close/ClosePointInTimeApiTests.cs:16]: SkipVersion
```suggestion
	[SkipVersion("<7.10.0", "Api introduced in 7.10.0")]
	public class ClosePointInTimeApiTests
```

**Review by Mpdreamz — APPROVED** [review]
LGTM small note on SkipVersion

File: tests/Tests/Search/PointInTime/Close/ClosePointInTimeApiTests.cs (hunk 1 above)
Comment (stevejgordon) [reply tests/Tests/Search/PointInTime/Close/ClosePointInTimeApiTests.cs:17]: Oops, yes! Thanks!

File: tests/Tests/Search/PointInTime/Close/ClosePointInTimeApiTests.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 namespace Tests.Search.PointInTime.Close
 {
+	[SkipVersion("<7.10.0", "Api introduced in 7.10.0")]
```
Comment (Mpdreamz) [inline tests/Tests/Search/PointInTime/Close/ClosePointInTimeApiTests.cs:16]: needs namespace imports :)

**Pull request merged**